### PR TITLE
Convert account pages to new query

### PR DESCRIPTION
**Pull request by chrismiceli**

### CODE REVIEW

**Comment by chrismiceli** [comment]
Will do more once I get confirmation I'm not completely on the wrong page. Available in my sandbox here: https://www.pgdp.org/~chrismiceli/c/

File: accounts/activate.php [hunk 1]
```diff
@@ -129,10 +129,10 @@
 // add ref to profile
 $refString=sprintf("
     UPDATE users SET u_profile=$profile->id WHERE id='%s' AND username='%s'
```
Comment (cpeel) [inline accounts/activate.php:131]: `u_profile = %d` and pull `$profile->id` into the sprintf() args.

**Comment by chrismiceli** [comment]
Merged in master, resolved merge conflicts, and updated my sandbox.

**Comment by cpeel** [comment]
Rather than do all files in one huge MR, let's do one directory's worth at a time. It makes it easier to review and merge in. Can we isolate this one to just the files in `accounts/` please?

**Comment by chrismiceli** [comment]
> 
> 
> Rather than do all files in one huge MR, let's do one directory's worth at a time. It makes it easier to review and merge in. Can we isolate this one to just the files in `accounts/` please?

Absolutely, sorry got a bit over zealous. Let me know which directory you want me to tackle next in a separate MR.

**Comment by cpeel** [comment]
This looks good. The branch is reporting merge conflicts however, please do an interactive rebase off of upstream and squash your changes down into one commit.

We might as well start on the tools directory, so perhaps `tools/authors/`.

**Comment by chrismiceli** [comment]
> 
> 
> This looks good. The branch is reporting merge conflicts however, please do an interactive rebase off of upstream and squash your changes down into one commit.
> 
> We might as well start on the tools directory, so perhaps `tools/authors/`.

Squashed to 1 commit and ensured no merge conflicts.

**Pull request merged**